content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -228,6 +228,13 @@ and Ready or completely terminated prior to launching or terminating another
 Pod. This option only affects the behavior for scaling operations. Updates are not
 affected.
 
+#### Minimum Ready Seconds
+
+`.spec.minReadySeconds` is an optional field that specifies the minimum number of seconds for which a newly
```

Comment:
You may want to insert the following line before line 233:

```
{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
```

Reply:
Done. Please let me know if it is at right location.